shells/android/java/arcs/crdt/parcelables/ParcelableCrdtType.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.crdt.parcelables
+
+import android.os.Parcelable
+import arcs.crdt.CrdtData
+import arcs.crdt.CrdtModel
+import arcs.crdt.CrdtOperation
+
+/** Enumeration of the parcelable [CrdtModel] types. */
+enum class ParcelableCrdtType(
+    /** [Parcelable.Creator] for the type's [ParcelableCrdtData] class. */
+    val crdtDataCreator: Parcelable.Creator<out ParcelableCrdtData<out CrdtData>>? = null,
```

Comment:
FMI: what does `out` do as a generic argument?

Reply:
`out` is similar to `? extends` in Java